test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2617,6 +2617,63 @@ def forward(self, *tensor_list):
         x = torch.randn(3, 4)
         self.run_test(EinsumModelTranspose(), input=(x,))
 
+    @unittest.skip("Enable this once ORT version is updated")
```

Comment:
could you add 1 or 2 tests in test/onnx/test_operators.py as well, to test the graph schema?